docs/wiki/ALZ-Contribution-Guide.md

```diff
@@ -126,6 +126,18 @@ Policy versioning follows the same protocol as built-in policies. More informati
 
 For policy deprecation, the process is documented in the [Azure Landing Zones - Deprecating Policies](./ALZ-Deprecated-Services.md) page.
 
+If a policy is part of an initiative, references to policies that are being deprecated should be removed. Policy initiatives are located in the [policySetDefinitions](../../src/resources/Microsoft.Authorization/policySetDefinitions/) folder. To find out if a policy is part of an initiative it is recommended to look up the policy definition in [AzAdvertiser](http://azadvertizer.com/) and check for association with initiatives. When identified, go into the necessary initiative and remove references to the definition. Locate the policy definition in the parameters of the initiative and remove reference:
+
+![Example policy def in initiative](./media/example-def-in-init.png)
+
+Also find it in the policyDefinitions and remove reference as well:
+
+![Example policy def in initiative 2](./media/example-def-in-init-2.png)
+
+When working within the policy files, to read parameters which are set at the top level of the policy definition a double escape is needed. So instead of using `[parameters('someParameter')]` within the policy effect, it should read as `[[parameters('someParameter')]`.
+
+When working with policies that are assigned by default, these are located under the [eslzArm/managementGroupTemplates/policyAssignments](../../eslzArm/managementGroupTemplates/policyAssignments) folder. References to policy definitions are done through the assignments, so if any amendments are done to default assigned policies, they should be amended here too. A wiki to default assignments can be found [in the wiki](./ALZ-Policies.md).
```

Comment:
Add into here on top of what you already have that the `eslzArm.json` file will need updating also if wanting to assign a new policy that is located here: https://github.com/Azure/Enterprise-Scale/blob/main/eslzArm/eslzArm.json

Reply:
Added to the guide

Reply:
I cant see this update for `eslzArm.json` around assignments?

Reply:
ignore me i can now